lib/ht_sip_validator/validation/checksums/well_formed.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require "ht_sip_validator/validation/base"
+
+module HathiTrust::Validation::Checksums
+  # validates that checksums values are well formed
+  class WellFormed < HathiTrust::Validation::Base
+    def validate
+      checksum_values = @sip.checksums.checksums.values
```

Comment:
the `.checksums.checksums` seems like it one of these two layers could use a different name, but it might only ever come up here.